packages/network/src/logic/Node.ts

```diff
@@ -107,6 +119,27 @@ export class Node extends EventEmitter {
         }
         this.metricsContext.addMetrics('node', this.metrics)
 
+        if (opts.signers !== undefined) {
+            this.receiptStore = new ReceiptStore(this.peerInfo.peerId)
+            this.receiptRequester = new ReceiptRequester({
```

Comment:
The receipt request could be conditionally enabled; namely if we are a miner node in a bounty. Actually might to need add logic that only certain streams are active for receipt requesting. Those would be the streams that we are getting paid to broker (i.e. part of bounty contract).